tests/integration_tests/configs/default.jsonnet

```diff
@@ -19,6 +19,7 @@
         'feehistory-cap': 100,
         'block-range-cap': 10000,
         'logs-cap': 10000,
+        'fix-revert-gas-refund-height': 1,
```

Comment:
test fail seems not related to this change, but due to the [deprecate](https://github.com/evmos/ethermint/commit/0f7bdceaa0b37ec0c823d8ca30f82bc3af275aac) change